src/extras/PMREMGenerator.js

```diff
@@ -611,29 +611,31 @@ ${_getEncodings()}
 #define ENVMAP_TYPE_CUBE_UV
 #include <cube_uv_reflection_fragment>
 
+vec3 getSample(float theta) {
+  vec3 axis = latitudinal ? poleAxis : cross(poleAxis, vOutputDirection);
+  if (all(equal(axis, vec3(0.0))))
+    axis = cross(vec3(0.0, 1.0, 0.0), vOutputDirection);
+  axis = normalize(axis);
```

Comment:
Ugh, I can't believe I missed this when I wrote it first: these three lines calculating the axis can be moved out of the loop entirely as they don't depend on theta. That'll speed things up a bit.

Reply:
hmm...
```glsl
axis = vec3( vOutputDirection.z, 0.0, - vOutputDirection.x );
```

Reply:
> hmm...
> 
> ```glsl
> axis = vec3( vOutputDirection.z, 0.0, - vOutputDirection.x );
> ```

Will this make any difference in speed? I feel using 'cross' is more clear and readable. *shrug* don't care that much.

Reply:
Gonna resolve for now. If ya'll feel strongly about the cross fix, I'll do another PR. I did the axis-opt.

Reply:
> 
> 
> > hmm...
> > ```glsl
> > axis = vec3( vOutputDirection.z, 0.0, - vOutputDirection.x );
> > ```
> 
> Will this make any difference in speed? I feel using 'cross' is more clear and readable. _shrug_ don't care that much.

In fact that CROSS will make it look "obfuscated" and run slower.

Reply:
I agree with @WestLangley. :-)

/ping @elalish

Reply:
I also have no strong opinion. May as well change it.

Reply:
Done